### ISSUE TITLE
quirks to convey to translators when this goes live

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#33]
Our plan is to implement an initial, fairly simple version of the translation utility and then add features and functionality later.  We're going to share this only with trusted translators initially.  There are a number of quirks that this initial version will have, and we should let them know about these.  This issue is the place to collect such quirks.

**Comment by jbphet** [comment]
There will be some strings that appear in the translation list that users probably won't be able to see in the sim, such as those related to branding, e.g. "Adapted from".

**Comment by aaronsamuel137** [comment]
Another quirk to convey: There will be cases where the old Java string had html in it. However, this won't work in the HTML5 version unless the string is explicitly an html string. In some cases, these strings are now MultilineStrings instead:

<img width="854" alt="screen shot 2015-09-10 at 4 23 58 pm" src="https://cloud.githubusercontent.com/assets/3193912/9802579/90b86b8e-57d8-11e5-9b8b-fc2309fcd8df.png">

In the case shown here they will need to replace the html with the "\n" character in order to get the string to work properly.

Basically, the rule of thumb they need to follow is this: If there is a "\n" in the English it means they can use "\n" to get a new line in the translation. If there is html in the English, it means they can use html in the translation. If there is no html in the English, it won't work if they have some in the translation.

**Comment by pixelzoom** [comment]
Perhaps we should fix these cases in the ported string files?

**Comment by ariel-phet** [comment]
@pixelzoom what would it take to make these fixes? Is it something an undergrad could take care of with some guidance?

**Comment by pixelzoom** [comment]
Write a script that looks through string files for HTML markup that consists solely of `<html>` and `<br>` tags, replaces the "<html>" and "</html>" with "" (empty string) and replaces "<br>" with "\n".  It would also be good to warn about other more complicated HTML markup, because it may not be supported in HTML5 sims.

**Comment by pixelzoom** [comment]
9/17/15 dev meeting:
• @aaronsamuel137 will discuss with @pixelzoom
• @samreid suggested just doing search-&-replace in IntelliJ

**Comment by aaronsamuel137** [comment]
I've removed all these strings. Assigning back to @ariel-phet to close or do something else here.

**Comment by ariel-phet** [comment]
Rosetta live, known quirks conveyed in documentation. Closing
